webapp/nuxt.config.js

```diff
@@ -47,10 +54,20 @@ export default {
   // Axios module configuration (https://go.nuxtjs.dev/config-axios)
   axios: {},
 
+  pwa: {
+    manifest: {
+      name: 'HTW: News List',
+      short_name: 'HTW: News List',
+      description: 'Lists News from a GraphQL Endpoint',
+    },
+  },
+
   apollo: {
     clientConfigs: {
       default: {
         httpEndpoint: 'https://news-list-backend.herokuapp.com',
+        authenticationType: 'Bearer',
```

Comment:
```suggestion
```
Is already the default: https://github.com/nuxt-community/apollo-module#advanced-configuration